### PR TITLE
Add chocolate-coon version

### PR DESCRIPTION
**Pull request by bbdjjtx**
Please review this @casey-collab

### CODE REVIEW

**Comment by bbdjjtx** [comment]
Please review this update @casey-collab

File: english/light-fire/sprinkler/basement-swim/dried-chocolate/chocolate-cocoon/chocolate-cocoon.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+As you wait for the chocolate to dry, you quickly fall asleep and start dreaming of being a kid again. You dream
```
Comment (casey-collab) [inline english/light-fire/sprinkler/basement-swim/dried-chocolate/chocolate-cocoon/chocolate-cocoon.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/light-fire/sprinkler/basement-swim/dried-chocolate/chocolate-cocoon/chocolate-cocoon.md (hunk 1 above)
Comment (casey-collab) [inline english/light-fire/sprinkler/basement-swim/dried-chocolate/chocolate-cocoon/chocolate-cocoon.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**